lib/start/src/bottom_stuff.dart

```diff
@@ -68,12 +78,13 @@ class BottomStuff extends HookWidget {
               onPressed: LoginLabels.goto(target),
               child: SizedBox(
                 width: double.infinity,
-                child: Text(text, style: spaced, textAlign: TextAlign.center),
+                child: Text(text,
+                    style: spaced.copyWith(fontSize: fontSize), textAlign: TextAlign.center),
               ),
             );
 
             Widget buttonStuff = Padding(
-              padding: const EdgeInsets.symmetric(vertical: 10),
+              padding: EdgeInsets.symmetric(vertical: bottomPadding),
```

Comment:
I'm wondering if the extra computation for this vertical space is worth it, since the main problem is with the lack of horizontal space.